cmd/datamon/cmd/label_get.go

```diff
@@ -0,0 +1,86 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/oneconcern/datamon/pkg/core"
+	"github.com/oneconcern/datamon/pkg/model"
+	"github.com/oneconcern/datamon/pkg/storage/gcs"
+
+	"github.com/spf13/cobra"
+	"golang.org/x/sys/unix"
+)
+
+var GetLabelCommand = &cobra.Command{
+	Use:   "get",
+	Short: "Get bundle info by label",
+	Long: `Performs a direct lookup of labels by name.
+Prints corresponding bundle information if the label exists,
+exits with ENOENT status otherwise.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		if params.repo.ContributorEmail == "" {
```

Comment:
Thought not enforced uniformly, we should allow read operations without contributor information being set.

Reply:
yes, this was removed altogether